extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -69,34 +69,30 @@ class AmpAccordion extends AMP.BaseElement {
           'Each section must have exactly two children. ' +
           'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
           'amp-accordion/amp-accordion.md. Found in: %s', this.element);
-      this.mutateElement(() => {
```

Comment:
This should at least also fix #3586 (probably more reliably). That fix wasn't great. We probably need a `markMyChildrenAsDirty` API that doesn't schedule a mutate.

CC @aghassemi